tests/fetch_body_test.go

```diff
@@ -657,7 +657,7 @@ Errors-to: dovecot-bounce@procontrol.fi
 X-original-sender: tss@iki.fi
 Precedence: bulk
 X-list: dovecot
-X-UID: 3                                                  
+X-UID: 3
```

Comment:
why remove whitespace in the end of the line? I think it's part of the test isn't it?

Reply:
Whitspace trimming madness. Will revert.

Reply:
Fixed.